lib/install.js

```diff
@@ -142,6 +143,15 @@ exports.install = function(targets, options) {
       return install(name, targets[name], opts);
     }))
     .then(function() {
+      var hooks = config.pjson.hooks;
+      if(hooks) {
+        if('postinstall' in hooks && typeof hooks.postinstall === 'string') {
+          System.config(config.getLoaderConfig());
+          return System.import(hooks.postinstall);
+        }
+      }
+    })
+    .then(function() {
       return saveInstall();
```

Comment:
Are you sure we don't want to run the hook after `saveInstall` so that the new configuration files have been saved already?